config/rbac/role.yaml

```diff
@@ -25,6 +25,7 @@ rules:
   resources:
   - namespaces
   verbs:
+  - delete
```

Comment:
I think you forgot to rebuild bundles after adding this in. CSV also needs that permission added.

Reply:
Before you regenerate, please also review the additional images file: https://github.com/integr8ly/integreatly-operator/blob/master/products/additional-images.yaml and ensure that the grafana image we are about to use is listed there. Thanks

Reply:
done . Commit: https://github.com/integr8ly/integreatly-operator/pull/3377/commits/93df92d4920c3ec2db0e72c8fe46e7ace393e43a
Thank you